### ISSUE TITLE
data-remote links that enter the DOM after page load act as normal links

### ISSUE THREAD

**Issue by existentialmutt** [ParamagicDev/mrujs#42]
#### Expected Behaviour
I should be able to add a data-remote link to the DOM and have it work the same as one that was present at page load.

#### Actual Behaviour
data-remote links added to the DOM after page load don't submit a fetch request and act like normal links.

Interestingly, remote forms that are added to the DOM after page load still seem to work.

#### Steps to Reproduce
See https://github.com/existentialmutt/cable_car_messages/tree/mrujs .  Start up the app and click on New Message, then "Cancel" .

tested with mrujs-0.3.0-beta.11

**Comment by ParamagicDev** [comment]
Ahhh yes! this a known issue! This on the todo-list to get fixed. The temporary workaround is once youve added the `data-remote` link to the dom, call `mrujs.restart()` this will re-run all queries.

The real reason this happens is i havent gotten around to `MutationObservers` for everything! The reason forms still work is because form `submit` events get intercepted at the document level. With links, theres no document level event to hook into. 

TLDR: Im working on it, in the meantime, anytime you add a `data-remote` link, restart mrujs like so:

`window.mrujs.restart()`

**Comment by existentialmutt** [comment]
Thanks @ParamagicDev , glad it's a known thing!  I confirmed `mrujs.restart()` does fix it so I'll use that as a workaround for now.
